software/digimindmap.yml

```
@@ -0,0 +1,12 @@
+name: Digimindmap
+website_url: https://ladigitale.dev/digimindmap/#/
+description: Create simple mindmaps.
```

As the source code is in French, I think this is needed?

```suggestion
description: Create simple mindmaps (documentation in French).
```

---

Thanks

---

> the source code is in French

The documentation is, but the source code is not ;) But It can be done! https://github.com/bnjbvr/rouille